content/learn/keyconcepts.adoc

```diff
@@ -0,0 +1,164 @@
+
+[id="helm-and-kustomize"]
+== Helm and Kustomize
+
+Two options are available that allow you to manage the lifecycle of Kubernetes applications you want to deploy with the validated patterns framework. The tools supported are `Kustomize` and `Helm`. 
+
+* *Kustomize*: Kustomize is a configuration management tool used to make declarative changes to application configurations and components and preserve the original base YAML files. Kustomize allows you to manage Kubernetes resource files in a declarative manner, using a set of layered configurations. This makes it easier to maintain, version, and apply changes to Kubernetes resources without having to modify the original YAML files directly.
+* *Helm*: Helm is a package manager for Kubernetes, which simplifies the process of defining, installing, and upgrading complex Kubernetes applications. It manages Kubernetes manifests using a packaging format called `Charts`.
+
+Helm is the preferred mechanism for deploying applications in the validated patterns framework, and detailed information about its usage follows.
+
+[id="about-helm"]
+== Helm in validated patterns
+
+Helm is the preferred tool for deploying applications within the validated patterns framework. Helm charts are packages that describe a set of Kubernetes resources ready for deployment, including customizable values for application deployment and functions for distributing charts and updates. A Helm chart consists of files structured to include chart metadata such as name and version, resource definitions, and supporting materials. This structure facilitates clear organization and management of Kubernetes resources.
+
+A minimal Helm chart includes: 
+ 
+----
+sample/
+├── Chart <1>
+├── templates <2>
+│   └── example.yaml
+└── values.yaml <3>
+----
+
+<1> The `Chart.yaml` file contains chart metadata, such as the name and version of the chart.
+<2> The `templates` directory contains files that define application resources such as
+deployments.
+<3> The `values.yaml` file contains default values for the chart.
+
+[id="about-helm-argocd"]
+== ArgoCD and Helm Integration
+
+ArgoCD integrates with Helm to provide a powerful GitOps-based deployment mechanism. The validated patterns framework uses ArgoCD and Helm to streamline application deployment by defining applications as Helm charts stored in Git repositories. ArgoCD is the tool of choice to apply the desired state of desired application to the target cluster environment. 
+
+ArgoCD automates the deployment and synchronization of these applications to OpenShift Container Platform clusters, ensuring consistency, reliability, and efficiency in managing Kubernetes applications. This integration supports automated, declarative, and version-controlled deployments, enhancing operational efficiency and maintaining application state across environments. The strength of ArgoCD is to help implement continuous deployment for cloud native applications.  
```

Comment:
Suggestion: May be we can rewrite this sentence just to remove the fluff " The strength of ArgoCD is to help implement continuous deployment for cloud native applications. "
to "ArgoCD helps implement continuous deployment for cloud-native applications."